python/ray/_raylet.pyx

```diff
@@ -1667,6 +1667,12 @@ cdef class CoreWorker:
             CCoreWorkerProcess.GetCoreWorker().RemoveLocalReference(
                 c_object_id)
 
+    def get_owner_address(self, ObjectRef object_ref):
+        cdef:
+            CObjectID c_object_id = object_ref.native()
+        return CCoreWorkerProcess.GetCoreWorker().GetOwnerAddress(
+                c_object_id).SerializeAsString()
```

Comment:
This function is added because locations of objects need to be passed to java, so that we can ensure locality in spark.

Reply:
Are you calling this from RayDP? This isn't a public API, so we should call it only from within Ray code (and pass the addresses to external libraries).

Reply:
Is this comment addressed?

Reply:
Well I added the developer api, I'm going to directly call it in `to_spark` now.